advanced/Templates/pihole-FTL.service.ini

```diff
@@ -0,0 +1,37 @@
+[Unit]
+Description=pihole-FTL Domain Name Server
+Documentation=man:pihole-FTL(8)
+After=remote-fs.target
+
+# Alternative based on https://wiki.debian.org/Hardening/Daemons#bind
+#   allows other services which rely on DNS to use pihole-FTL, but means
+#   breaking if config relies on remote-fs
+# After=network.target
```

Comment:
Its disabled because I'm not entirely sure what the interactions between `After=remote-fs.target` and `After=network.target` would be, so I just left the dependencies as a mirror of the init script.

Reply:
After thinking about it some more, it should be fine since multiple `After`'s are safe. The problem would be between `After=remote-fs.target` and `Before=nss-lookup.target` since presumably remote mounts would require DNS in some cases.